.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
+}
```

Comment:
Do we need this to set the TypeScript version? Can VSCode auto-detect it?

Reply:
Oh whoops, added this by accident. Will remove 👍